PublicSDKAPITester/RevenueCatAPI.m

```diff
@@ -10,6 +10,7 @@
 #import "RCPurchases.h"
 
 @import StoreKit;
+@import PurchasesCoreSwift;
```

Comment:
right, this is what I was talking about here https://github.com/RevenueCat/purchases-ios/pull/585#discussion_r658828948, we need the `PurchasesCoreSwift` import since it's no longer exposed through Purchases

Reply:
Once we're ready to pull the plug on the migration, though, we should be able to remove this import altogether without anything breaking. That's how we know we've succeeded. 
Perhaps we could also add  a `todo` comment here explaining that?

Reply:
https://github.com/RevenueCat/purchases-ios/issues/593